src/com/android/calendar/Utils.java

```diff
@@ -682,7 +682,12 @@ public static boolean getHideDeclinedEvents(Context context) {
 
     public static int getDaysPerWeek(Context context) {
         final SharedPreferences prefs = GeneralPreferences.getSharedPreferences(context);
-        return prefs.getInt(GeneralPreferences.KEY_DAYS_PER_WEEK, 7);
+        return Integer.valueOf(prefs.getString(GeneralPreferences.KEY_DAYS_PER_WEEK, "7"));
+    }
+
+    public static int getMDaysPerWeek(Context context) {
+        final SharedPreferences prefs = GeneralPreferences.getSharedPreferences(context);
+        return Integer.valueOf(prefs.getString(GeneralPreferences.KEY_MDAYS_PER_WEEK, "7"));
```

Comment:
This looks like we get the same functionality twice. Would it make sense to merge this, or add a second configuration option?